### PR TITLE
toil(bandit): silence request_without_timeout

### PR DESCRIPTION
**Pull request by ben-alkov**
https://issues.redhat.com/browse/STONEBLD-941

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- N/A New code has type annotations
- [x] Docs updated (if applicable)
- [x] Docs links in the code are still valid (if docs were updated)

### CODE REVIEW

**Comment by ben-alkov** [comment]
@crew:
I'm not sure what the timeout should be (I put 5s because that's what the bandit example had :grin:)...

**Comment by chmeliik** [comment]
> @crew: I'm not sure what the timeout should be (I put 5s because that's what the bandit example had grin)...

I'd go with ~30-60 seconds, and +1 to make it configurable

File: cachi2/core/package_managers/general.py [hunk 1]
```diff
@@ -20,10 +21,11 @@ def download_binary_file(url, download_path, auth=None, insecure=False, chunk_si
     :param int chunk_size: Chunk size param for Response.iter_content()
     :raise FetchError: If download failed
     """
+    timeout = get_config().gomod_download_max_tries
```
[the remark this answers is not in the document]
Comment (ben-alkov) [reply cachi2/core/package_managers/general.py:24]: 😑

[the remark this answers is not in the document]
Comment (lkolacek) [reply cachi2/core/package_managers/general.py:24]: makes sense to change this to `requests_timeout`

**Comment by brunoapimentel** [comment]
We also need to update the docs: https://github.com/containerbuildsystem/cachi2#available-configuration-parameters

**Review by taylormadore — APPROVED** [review]

**Review by lkolacek — APPROVED** [review]
LGTM :+1:

**Pull request merged**